build/scripts/build.ps1

```diff
@@ -25,7 +25,7 @@ function Print-Usage() {
 }
 
 function Run-Build() {
-    $buildArgs = "/v:m /m"
+    $buildArgs = "/v:m /m /flp:v=diag"
```

Comment:
You don't, local testing change escaped when I `git commit -am`-ed. Will revert.